transform/mattermost-analytics/models/staging/mm_telemetry_prod/stg_mm_telemetry_prod__performance_events.sql

```diff
@@ -0,0 +1,73 @@
+{{
+    config({
+        "tags":"hourly",
+    })
+}}
+
+WITH MM_TELEMETRY_PROD AS 
+(SELECT _dbt_source_relation	  AS _dbt_source_relation	
+ , context_page_referrer          AS context_page_referrer
+ , channel                        AS channel
+ , context_app_namespace          AS context_app_namespace
+ , id                             AS id
+ , user_actual_id                 AS user_actual_id
+ , context_library_name           AS context_library_name
+ , type                           AS type
+ , context_app_version            AS context_app_version
+ , user_actual_role               AS user_actual_role
+ , context_page_url               AS context_page_url
+ , context_os_name                AS context_os_name
+ , context_page_title             AS context_page_title
+ , anonymous_id                   AS anonymous_id
```

Comment:
IIRC correctly, `anonymous_id` is always added with a default value. Perhaps it's not needed.

Reply:
more about `anonymousId`: https://www.rudderstack.com/docs/event-spec/standard-events/identify/

Reply:
I will remove the fields in the next iteration, after working with Harrison.